packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -73,7 +73,7 @@ class CupertinoSwitch extends StatefulWidget {
   /// change state until the parent widget rebuilds the switch with the new
   /// value.
   ///
-  /// If null, the switch will be displayed as disabled.
+  /// If null, the switch will be displayed as disabled with an opacity of [kCupertinoSwitchDisabledOpacity]
```

Comment:
The reference to the constant here won't work because the constant is private. Please consider a more ambiguous wording that allows us to avoid referencing the constant, e.g., "If null, the switch will be displayed as disabled, which has a reduced opacity."

Reply:
Made the change